src/index.js

```diff
@@ -112,28 +122,71 @@ class EHoldings extends Component {
         </Route>
       )
       : (
-        <CommandList commands={defaultKeyboardShortcuts}>
-          <KeyShortcutsWrapper focusSearchField={this.focusSearchField}>
-            <Route path={rootPath} component={ApplicationRoute}>
-              <Switch>
-                <Route path={`${rootPath}/providers/:providerId`} exact component={ProviderShow} />
-                <Route path={`${rootPath}/providers/:providerId/edit`} exact component={ProviderEdit} />
-                <Route path={`${rootPath}/packages/new`} exact component={PackageCreate} />
-                <Route path={`${rootPath}/packages/:packageId`} exact component={PackageShow} />
-                <Route path={`${rootPath}/packages/:packageId/edit`} exact component={PackageEdit} />
-                <Route path={`${rootPath}/titles/new`} exact component={TitleCreate} />
-                <Route path={`${rootPath}/titles/:titleId`} exact component={TitleShow} />
-                <Route path={`${rootPath}/titles/:titleId/edit`} exact component={TitleEdit} />
-                <Route path={`${rootPath}/resources/:id`} exact component={ResourceShow} />
-                <Route path={`${rootPath}/resources/:id/edit`} exact component={ResourceEdit} />
-                <Route path={`${rootPath}/notes/new`} exact component={NoteCreate} />
-                <Route path={`${rootPath}/notes/:noteId`} exact component={NoteView} />
-                <Route path={`${rootPath}/notes/:id/edit`} exact component={NoteEdit} />
-                <Route path={`${rootPath}/`} exact component={SearchRoute} />
-              </Switch>
-            </Route>
-          </KeyShortcutsWrapper>
-        </CommandList>
+        <>
+          <AppContextMenu>
```

Comment:
I think we should move this to a separate component like `EHoldingsAppContext` to not make root component too large